nats.go

```diff
@@ -3505,15 +3535,17 @@ func (nc *Conn) newRespInbox() string {
 	if nc.respMap == nil {
 		nc.initNewResp()
 	}
-	var b [respInboxPrefixLen + replySuffixLen]byte
-	pres := b[:respInboxPrefixLen]
-	copy(pres, nc.respSub)
+
```

Comment:
Looks like respInboxPrefixLen (and replySuffixLen) may no longer be used? If so, will have to remove.